### PR TITLE
Add center, backgroundColor and transform properties

### LINKED ISSUE #3: backgroundColor, center, transform
Additional animatable properties for `UIView`.

### PR DESCRIPTION
**Pull request by delebedev**
Resolves #3.

### CODE REVIEW

File: ReactiveAnimation/View.swift [hunk 1]
```diff
@@ -56,6 +56,20 @@ public struct RAN {
 		return viewProperty { self.animator?.bounds = $0 }
 	}
 
+    #if os(iOS)
+    public var center: ViewProperty<CGPoint> {
+        return viewProperty { self.animator?.center = $0 }
+    }
+    
+    public var backgroundColor: ViewProperty<CGColor> {
+        return viewProperty { self.animator?.layer.backgroundColor = $0 }
```
Comment (jspahrsummers) [inline ReactiveAnimation/View.swift]: Can this use `UIView.backgroundColor` instead?

**Comment by jspahrsummers** [comment]
Awesome, thanks! :sparkles:

Just the one note.

File: ReactiveAnimation/View.swift (hunk 1 above)
Comment (delebedev) [inline ReactiveAnimation/View.swift]: I thought backgroundColor is not directly animatable, but was wrong. Fixed that

**Comment by jspahrsummers** [comment]
:shipit:

**Pull request merged**